instrumentation/jms/src/main/java/brave/jms/MessageParser.java

```diff
@@ -58,6 +59,16 @@ static boolean isQueue(@Nullable Destination destination) {
     return isQueue;
   }
 
+  /**
+   * Similar to other properties, {@code null} should be expected even if it seems unintuitive.
```

Comment:
I probably should have just given a suggestion than request ;) I didn't mean channel name, just a line or so in `isQueue` above the `getQueueName` null check saying while the spec doesn't seem to define it, in practice implementations that implement both interfaces return only one of the names for which they are active.